etc/avocado/avocado.conf

```diff
@@ -38,6 +38,14 @@ utf8 =
 # Keep job temporary files after jobs (useful for avocado debugging)
 keep_tmp_files = False
 
+[remoter.behavior]
+# Reject unknown SSH host keys.
+# 'False' will leave you wide open to man-in-the-middle attacks!
+# 'True' will only work with RSA keys (due to a bug in Paramiko).
+# If using 'True', accept the remote host key fingerprint by using:
```

Comment:
Please add the bug reference here. Actually how about adding `selftest`, which would use `urllib` to check if the issue is still open? Github is quite flexible and it's easy to do such verification. That way we'd avoid keeping it unnecessarily long. It should marked as skip when url is inaccessible.

PS: It'd be really cool if you implement it as `avocado.utils` so we can reuse it everywhere github is used as reference. Anyway don't spend too much time on it, only if it's really simple (and I think it is)